src/Util/FileHasher.php

```diff
@@ -40,6 +40,18 @@ public function hashFiles(array $files): string
         return $configHash;
     }
 
+    public function resolvePath(string $filePath): string
```

Comment:
Moved here so that it could be used both by the file changed detector and the dependency cache